test/oats/configs/grafana-datasources.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: 1
+
+datasources:
```

Comment:
I think this file is also for Grafana's UI, if we don't have any dashboards perhaps we don't need it.

Reply:
It's useful for "make oats-test-debug" to see what's going on.